src/js/components/Notification/Notification.js

```diff
@@ -45,38 +94,74 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   ]);
 
   const { icon: CloseIcon } = theme.notification.close;
-  const { icon: StatusIcon, color } = theme.notification[status];
+  const { icon: StatusIcon, background, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const { direction, truncate } = toast
+    ? theme.notification.toast
+    : theme.notification;
+
+  const TextWrapper = direction === 'row' ? Text : Box;
+  const textWrapperProps = direction === 'row' ? { truncate } : {};
+
+  // notification is built with two child boxes that contain:
+  // 1. icon + text (wrapped in button when clickable)
+  // 2. close button
+  // pad needs to be applied to the child boxes to ensure pad is included
+  // in the clickable region, but we don't want to apply extra padding
+  // between the icon + text and the button.
+  const { pad } = theme.notification.container;
+  let textPad;
+  let closeButtonPad;
+  if (onClose) [textPad, closeButtonPad] = adaptThemeStyle(pad, theme);
+  else textPad = pad;
+
   let content = (
     <Box
       {...theme.notification.container}
       {...(toast ? { ...theme.notification.toast.container } : {})}
+      background={
```

Comment:
what about building something like `callerProps` that captures properties passed in and then just expanding `{...callerProps}`?

Reply:
This isn't handling properties passed in -- it's just using theme values. I had destructured `background` above in the file